packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js

```diff
@@ -284,16 +309,43 @@ export default class QuanticStandaloneSearchBox extends NavigationMixin(
 
   onFocus() {
     this.showSuggestions();
+    this.adjustTextAreaHeight();
   }
 
   onBlur() {
     this.hideSuggestions();
+    this.collapseTextArea();
+  }
+
+  onTextAreaInput() {
+    this.adjustTextAreaHeight();
+  };
+
+  adjustTextAreaHeight() {
+    if (!this.textarea) {
+      return;
+    }
+    this.input.value = this.input.value.replace(/\n/g, '');
```

Comment:
nice touch!

Reply:
Yea I dunno if it'll be a problem in the future or not, it doesn't allow you to input structured text, it eats up your line breaks, but the line breaks also break the display of the searchbox when it is collapsed.